lib/jake/number.ex

```diff
@@ -2,6 +2,47 @@ defmodule Jake.Number do
   def gen(spec) do
     min = Map.get(spec, "minimum", -9_007_199_254_740_991)
     max = Map.get(spec, "maximum", 9_007_199_254_740_991)
-    StreamData.integer(min..max)
+    exclusive_min = Map.get(spec, "exclusiveMinimum", false)
```

Comment:
this seems rather complicated and not sure if it's necessary.

for starter, I would move the generator to StreamUtil,

```elixir
def number(min, max) do
  StreamData.one_of([
   StreamData.float(..),
   StreamData.integer(..)
  ])
end
```

would try to de-duplicate the code between number & integer. I think the general logic is same, just that some function/constant would be different. For exclusive min max, we could just write a filter that filters out both values like

```elixir
def filter(list) do
  StreamData.filter(&Enum.member?(list, &1))
end
```

Reply:
But how do we differentiate between integer and float stream here

```
Jake.StreamUtil.number(1,22) |> Enum.take(10)
[1.0, 1.0, 19.375, 22.0, 12.8125, 22.0, 22.0, 15, 8.21875, 8]
```

Reply:
why do we need to differentiate? Both integer and float are acceptable as a number?

> The number type is used for any numeric type, either integers or floating point numbers.

Reply:
Sorry, I thought you meant merging those 2